src/main/scala/no/ndla/listingapi/service/CoverValidator.scala

```diff
@@ -99,6 +101,13 @@ trait CoverValidator {
       }
     }
 
+    private def validateTheme(name: domain.ThemeName): Option[ValidationMessage] = {
```

Comment:
Nå husker hva denne var godt for. Ved PUT/POST kall, så validerer denne at noen ikke har oppdatert til en theme som ikke er lov. 
Den andre controller sjekken er for GET.